sdk/cosmos/azure_data_cosmos/CHANGELOG.md

```diff
@@ -1,12 +1,13 @@
 # Release History
 
-## 0.23.0 (Unreleased)
+## 0.22.1 (Unreleased)
 
 ### Features Added
 
 ### Breaking Changes
 
 ### Bugs Fixed
+* Fixed a publishing issue that caused the `key_auth` feature to be omitted. [#2241](https://github.com/Azure/azure-sdk-for-rust/issues/2241)
```

Comment:
Good idea to install markdown lint, which would've cause this:

```suggestion

* Fixed a publishing issue that caused the `key_auth` feature to be omitted. ([#2241](https://github.com/Azure/azure-sdk-for-rust/issues/2241))
```

Also nit: we tend to put bug references in parens, but not a big deal as long as you're consistent in your CHANGELOG.